keyboards/jj40/keymaps/waples/keymap.c

```diff
@@ -59,11 +50,11 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   _______, KC_F9, KC_F10, KC_F11, _______, _______, _______, _______, _______, _______, _______, _______ \
 ),
 
-[_DUAL] =  LAYOUT_ortho_4x12( \
-  RESET,   _______, _______, _______, _______, QWERTY,  GAME,    _______, _______, _______, BL_BRTG, RESET, \
-  _______, KC_MPRV, KC_MSTP, KC_MPLY, KC_MNXT, AG_NORM, AG_SWAP, _______, _______, _______, BL_TOGG, _______, \
-  _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, TG_NKRO, _______, _______, _______, _______, _______, \
-  _______, _______, _______, _______, _______, _______, DVORAK,  _______, _______, _______, _______, _______ \
+[_DUAL] =  KEYMAP( \
```

Comment:
We changed the layout macro naming convention around June of last year, to `LAYOUT` with an optional suffix, depending on the keyboard's codebase.

You need to change Line 53 as follows:

```suggestion
[_DUAL] =  LAYOUT_ortho_4x12( \
```

Reply:
changed